typings/index.d.ts

```diff
@@ -6,7 +6,7 @@ export interface DownshiftState<Item> {
   highlightedIndex: number | null
   inputValue: string | null
   isOpen: boolean
-  selectedItem: Item
+  selectedItem: Item | null
```

Comment:
if `null` is passed in the props, then `selectedItem` in the state object will always be `null`.